cmd/ddev/cmd/config.go

```diff
@@ -466,6 +470,18 @@ func handleMainConfigArgs(cmd *cobra.Command, _ []string, app *ddevapp.DdevApp)
 		app.HostDBPort = hostDBPortArg
 	}
 
+	if cmd.Flag("nfs-mount-enabled").Changed {
+		if v, _ := cmd.Flags().GetBool("nfs-mount-enabled"); v {
+			app.SetPerformanceMode(types.PerformanceModeNFS)
+		}
+	}
+
+	if cmd.Flag("mutagen-enabled").Changed {
+		if v, _ := cmd.Flags().GetBool("mutagen-enabled"); v {
+			app.SetPerformanceMode(types.PerformanceModeMutagen)
```

Comment:
```suggestion
			app.SetPerformanceMode(types.PerformanceModeMutagen)
						dirty = true

```

Reply:
We don't have this flag in config, config-global only.

Reply:
I don't understand the comment. In v1.21.6 we do in fact have mutagen and nfs per-project configuration.

Reply:
Talking about the dirty variable. This is only done in the global config with the dirty variable.